go/api/glide_cluster_client.go

```diff
@@ -372,3 +372,46 @@ func (client *GlideClusterClient) EchoWithOptions(echoOptions options.ClusterEch
 	}
 	return createClusterSingleValue[string](data), nil
 }
+
+// Returns UNIX TIME of the last DB save timestamp or startup timestamp if no save was made since then.
+// The command is routed to a random node by default, which is safe for read-only commands.
```

Comment:
Route is a mandatory argument there.
We need an implementation without route too.